$Header: /home/x_cvs/mit/server/ddx/x386/etc/stdlib.diff,v 1.1 1992/08/14 07:57:01 dawes Exp $

 This patch is required for /usr/include/stdlib.h when building X386
 on 386BSD.

diff -cb stdlib.h.dist stdlib.h
*** stdlib.h.dist	Mon Jul 13 13:24:09 1992
--- stdlib.h	Wed Jul 29 16:42:59 1992
***************
*** 102,108 ****
  #endif /* not ANSI */
  
  #if !defined(_ANSI_SOURCE) && !defined(_POSIX_SOURCE)
! void	*alloca __P((size_t));	/* built-in for gcc */
  extern	 char *optarg;			/* getopt(3) external variables */
  extern	 int optind;
  extern	 int opterr;
--- 102,112 ----
  #endif /* not ANSI */
  
  #if !defined(_ANSI_SOURCE) && !defined(_POSIX_SOURCE)
! #if defined(alloca) && (alloca == __builtin_alloca)
! void	*alloca __P((int));	/* built-in for gcc */
! #else
! void	*alloca __P((size_t));
! #endif /* __GNUC__ */
  extern	 char *optarg;			/* getopt(3) external variables */
  extern	 int optind;
  extern	 int opterr;
